lib/ProductOpener/Display.pm

```diff
@@ -6964,19 +6983,6 @@ sub display_field($$) {
 		
 		# Separate To-Do and Done Status
 		if ($field eq 'states') {
```

Comment:
What's in the "if ($field eq 'states') {" condition should now be moved above, (after you set the values $to_do_status and $done_status), as you already test for that condition above.